test/e2e/prune_test.go

```diff
@@ -320,9 +322,6 @@ var _ = Describe("Podman prune", func() {
 		create.WaitWithDefaultTimeout()
 		Expect(create.ExitCode()).To(Equal(0))
 
-		// Adding images should be pruned
-		podmanTest.BuildImage(pruneImage, "alpine_bash:latest", "true")
```

Comment:
Why are you removing this?

Reply:
Build image with layers is not implemented by a remote client. And for this use case it simply added a bunch of more images using build image and checks if they are pruned with system prune command. However the addition of BB image here https://github.com/containers/libpod/pull/6318/files#diff-9e3f37b4287d658db5658028b861b77bL319 and validating it pruned here https://github.com/containers/libpod/pull/6318/files#diff-9e3f37b4287d658db5658028b861b77bL351
addresses the use case hence the test need not be skipped for that matter and is still accounts as a valid test.